tests/mime/mime-dec-parse-full-msg-test01/test.yaml

```diff
@@ -0,0 +1,48 @@
+# *** Add configuration here ***
+
```

Comment:
We usually ask folks to remove this. :P

Reply:
Ok.

Why is it not removed automatically ?